news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md

```diff
@@ -0,0 +1,153 @@
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 01st to Sunday 07th of April 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+The tests and debug phase of the 1.7.6 release has started: 134 issues have been created on the week after the code freeze. Some developers are already testing it on their infrastructure, opening issues, and submiting pull request, and that is AWESOME!!!
+
+Please continue like that, thank you very much for your feedback.
+
+Oh, and this week, I went to the first edition of the PSconnect event in Lyon, France, with a few people of the PrestaShop company: business managers, product managers, our CEO and our CTO. That was a very interesting moment to discuss and collect feedback about the roadmap. Merchants, partners, agencies and freelancers confirmed that the project's roadmap is on good tracks. There is a very good feedback about 1.7.5, with many projects on production and scheduled migrations.
+
+Find the next PrestaShop event in your area: [https://www.prestashop.com/en/events](https://www.prestashop.com/en/events)
+
+
+## A quick update about PrestaShop's GitHub issues and pull requests:
+
+- [134 new issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+created%3A2019-04-08..2019-04-14) have been created in the project repositories;
+- [63 issues have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+closed%3A2019-04-08..2019-04-14), including [12 fixed issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+label%3Afixed+closed%3A2019-04-08..2019-04-14) on the core;
+- [74 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2019-04-08..2019-04-14) in the project repositories;
+- [46 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2019-04-08..2019-04-14), including [43 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2019-04-08..2019-04-14).
+
+
+
+## Code changes in the "1.7.6.x" branch (for v1.7.6.0)
+
+### Core
+
+* [#13246](https://github.com/PrestaShop/PrestaShop/pull/13246): Add missing hook description, by [@eternoendless](https://github.com/eternoendless)
+* [#13256](https://github.com/PrestaShop/PrestaShop/pull/13256): Fixes to Customer subdomain & controller. Thank you [@sarjon](https://github.com/sarjon)
+* [#13266](https://github.com/PrestaShop/PrestaShop/pull/13266): Merge 1.7.6.x to develop - 10/04/2019, by [@matks](https://github.com/matks)
```

Comment:
This PR targeted the develop branch https://github.com/PrestaShop/PrestaShop/pull/13266

Also I think we can remove it as this is a "transfer PR" whose only goal is to retrieve commits from 1.7.6.x to develop

Reply:
moved, thanks.